packages/grafbase-sdk/CHANGELOG.md

```diff
@@ -1,73 +1,78 @@
 # Changelog
 
-## [0.4.0] - Thu Aug 03 2023
+## [0.5.0] - Tue Aug 08 2023
+
+[CHANGELOG](changelog/0.5.0.md)
+
+## [0.4.0] - Tue Aug 08 2023
 
 [CHANGELOG](changelog/0.4.0.md)
 
-## [0.3.0] - Fri Jul 21 2023
+## [0.3.0] - Wed Aug 02 2023
```

Comment:
What's with all these date changes?

Reply:
that's a very a good question ... I just ran `npm run bump-minor` ... reverting those.

Reply:
I have a suspicion though